1.12-rc2 fails to start on FreeBSD (under VT console)

Regressed by #9105.

```
$ sway -dc /dev/null
[...]
00:00:00.058 [INFO] [wlr] [types/wlr_linux_drm_syncobj_v1.c:447] Linux sync_file unavailable, disabling linux-drm-syncobj-v1
00:00:00.058 [ERROR] [sway/server.c:300] Failed to create linux-drm-syncobj v1
```

[Debug log](https://github.com/user-attachments/files/26663450/sway.log)

--- a/sway/server.c
+++ b/sway/server.c
@@ -295,11 +295,8 @@ bool server_init(struct sway_server *server) {
 	if (wlr_renderer_get_drm_fd(server->renderer) >= 0 &&
 			server->renderer->features.timeline &&
 			server->backend->features.timeline) {
-		if (!wlr_linux_drm_syncobj_manager_v1_create(server->wl_display, 1,
-				wlr_renderer_get_drm_fd(server->renderer))) {
-			sway_log(SWAY_ERROR, "Failed to create linux-drm-syncobj v1");
-			return false;
-		}
+		wlr_linux_drm_syncobj_manager_v1_create(server->wl_display, 1,
+				wlr_renderer_get_drm_fd(server->renderer));
 	}
 
 	server->allocator = wlr_allocator_autocreate(server->backend,

